bedrock/settings/base.py

```diff
@@ -1466,6 +1493,12 @@ def facebook_tab_url_lazy():
 # Optimize.ly project code
 OPTIMIZELY_PROJECT_ID = None
 
+# Fx Accounts Relier Service Client ID
+# TODO: Update when available! Currently using creds for demo5 & URLs for FxA dev
```

Comment:
So is there an IT bug that block this merging? Should we add a blocking comment to this PR with a link to the bug?